sass/themes/schemas/components/light/_switch.scss

```diff
@@ -402,33 +416,86 @@ $bootstrap-switch: extend(
 
 /// Generates an indigo switch schema.
 /// @type {Map}
-/// @prop {Map} thumb-off-color [color: ('gray', 600)] - The color of the thumb when the switch is off.
-/// @prop {Map} thumb-disabled-color [color: ('gray', 300)] - The color of the thumb when the switch is disabled.
+/// @prop {Map} label-color [color: ('gray', 800)] - The color of the switch label.
+/// @prop {Map} label-invalid-color [color: ('gray', 800)] - The color of the switch label in invalid state.
+/// @prop {Map} label-disabled-color [color: ('gray', 900, .2)] - The color of the switch label when the switch is disabled
+/// @prop {Map} thumb-on-color [contrast-color: ('gray', 900)] - The color of the thumb when the switch is on.
+/// @prop {Map} track-on-hover-color [color: ('primary', 400)] - The color of the track when the switch is on and hovered.
+/// @prop {Map} thumb-off-color [color: ('gray', 500)] - The color of the thumb when the switch is off.
+/// @prop {Map} thumb-disabled-color [color: ('gray', 900, .15)] - The color of the thumb when the switch is disabled.
+/// @prop {Map} thumb-on-disabled-color [contrast-color: ('gray', 900, .4)] - The color of the thumb when the switch is on and disabled.
+/// @prop {Map} track-on-disabled-color [color: ('primary', 400, .5)] - The color of the track when the switch is on and disabled.
 /// @prop {Map} border-color [color: ('gray', 600)] - The border color of the switch.
```

Comment:
I know you didn't change this one but the value is different from the map it should be 500